<launch>
    <rosparam file="$(find mimi_voice_control)/config/voice_cmd.yaml" command="load" ns="voice_cmd" />
    <node pkg="mimi_voice_control" name="vc_basic" type="vc_basic.py" />
</launch>
